openwpm/config.py

```diff
@@ -120,15 +120,30 @@ class ManagerParams(DataClassJsonMixin):
         default=Path.home() / "openwpm",
         metadata=DCJConfig(encoder=path_to_str, decoder=str_to_path),
     )
+    """The directory into which screenshots and page dumps will be saved"""
     log_path: Path = field(
         default=Path.home() / "openwpm" / "openwpm.log",
         metadata=DCJConfig(encoder=path_to_str, decoder=str_to_path),
     )
+    """The path to the file in which OpenWPM will log. The
+    directory given will be created if it does not exist."""
     testing: bool = False
+    """A platform wide flag that can be used to only run certain functionality
+    while testing. For example, the Javascript instrumentation"""
     memory_watchdog: bool = False
+    """- It is mostly useful for long running cloud crawls"""
```

Comment:
```suggestion
    """A watchdog that tries to ensure that no Firefox instance takes up too much memory. It is mostly useful for long running cloud crawls"""
```
Sorry for being imprecise with my feedback.

Reply:
I'm very happy you are getting something out of this as well!